quantecon/game_theory/game_generators/tests/test_bimatrix_generators.py

```diff
@@ -112,7 +112,7 @@ def test_seed(self):
     @raises(ValueError)
     def test_raises_value_error_too_large_inputs(self):
         n, k = 100, 50
-        g = tournament_game(n, k)
+        g = tournament_game(n, k)  # noqa: F841
```

Comment:
I would rather remove "`g =`" (than adding "`# noqa: F841`"). (The same applies to other places.)

Reply:
OK, will do that instead. I thought of adding `# noqa: F841` for consistency with the rest of the test cases.